src/sql/base/browser/ui/table/tableDataView.ts

```diff
@@ -63,6 +64,19 @@ function defaultFilter<T extends Slick.SlickData>(data: T[], columns: Filterable
 			});
 		}
 	});
+	const collapsibleTable = columns.find(c => c.formatter === expandableColumnFormatter);
+	if (collapsibleTable) {
+		filteredData = filteredData.filter((item) => {
+			let parent = data[item.parent];
+			while (parent) {
+				if (!parent._collapsed) {
+					return false;
```

Comment:
I'm not sure I understand this - if I'm reading this correctly isn't this filtering out the child elements from the data who have parents that are collapsed? But then shouldn't this be returning false if it finds a parent that's collapsed and default to true otherwise?

Reply:
Fixed the variable name. This code will make more sense.